src/style.css

```diff
@@ -1,3 +1,67 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the 'Roboto' font is correctly loaded in your HTML file. This is important for the font-family to render as expected.